### DISCUSSION TITLE
Disable Print Width for specific files

### DISCUSSION THREAD

**Discussion by cap1024** [prettier/prettier#16479] [Q&A] [answered]
Hello!

I need to prevent splitting long lines into two lines for specific files only without modifying these files.

How can I do this?

**Comment by kachkaev** [comment, marked as answer]
👋 @cap1024! I guess you can use overrides in your configuration filess:

```js
{
  "someGlobalPrettierOption": 42,
  "anotherGlobalPrettierOption": "xyz",
  "overrides": [
    {
      "files": "**/glob/to/your/files",
      "options": {
        "printWidth": 1000000
      }
    }
  ]
}
```

- https://prettier.io/docs/en/configuration.html#configuration-overrides
- https://prettier.io/docs/en/options.html#print-width
- https://code.visualstudio.com/docs/editor/glob-patterns

Please share your recipe with others, if it works!

**Reply by cap1024** [reply]
@kachkaev, thanks a lot, it works fine.
